### PR TITLE
[RAFlow] Checkboxes for Pets/Vehicle section

### PR DESCRIPTION
**Pull request by abataub**

### CODE REVIEW

File: webclient/js/elems/raflow.js [hunk 1]
```diff
@@ -58,6 +58,41 @@ $(document).on('click', '#ra-form #progressbar a', function () {
...
 });
 
+// lockOnGrid
+// Lock grid if chebox is unchecked(false). Unlock grid if checkbox is checked(true).
+// Lock grid when there is no record in the grid.
+window.lockOnGrid = function (gridName) {
+    console.log("lockOnGrid");
+    var isChecked = $("#" + gridName + "_checkbox")[0].checked;
+    var recordsLength = w2ui[gridName].records.length;
+    console.log(isChecked);
```
Comment (sudip-aubergine) [inline webclient/js/elems/raflow.js]: remove/comment these console statements if those are working fine

File: webclient/js/elems/raflow.js [hunk 2]
```diff
@@ -974,25 +1016,7 @@ window.loadRAPetsGrid = function () {
...
-                              '</label>'+
-                              '</div>';
-                            return html;
-                        }
-                    },
-                ]
-            },
-            multiSelect: false,
-            style: 'border: 1px solid black; display: block;',
+            style: 'border: 0px solid black; display: block;',
```
Comment (sudip-aubergine) [inline webclient/js/elems/raflow.js]: add `multiSelect: false` setting back to the code

File: webclient/js/elems/raflow.js [hunk 3]
```diff
@@ -1187,6 +1215,10 @@ window.getVehicleGridInitalRecord = function (BID, BUD, previousFormRecord) {
     return defaultFormData;
 };
 
+window.disableVehiclesGrid = function () {
```
Comment (sudip-aubergine) [inline webclient/js/elems/raflow.js]: do we need this?

File: webclient/js/elems/raflow.js (hunk 1 above)
Comment (abataub) [reply webclient/js/elems/raflow.js]: I need it for now.

Comment (sudip-aubergine) [reply webclient/js/elems/raflow.js]: :) :+1:

File: webclient/js/elems/raflow.js (hunk 2 above)
Comment (abataub) [reply webclient/js/elems/raflow.js]: ok

File: webclient/js/elems/raflow.js (hunk 3 above)
Comment (abataub) [reply webclient/js/elems/raflow.js]: About to remove.

**Pull request merged**